app/models/recipe.rb

```diff
@@ -1,3 +1,10 @@
 class Recipe < ApplicationRecord
    validates :name, :complexity, :genre, presence: true
+
+   has_many :recipe_ingredients
+   has_many :ingredients, through: :recipe_ingredients
+
+   def find_recipe_ingredients(recipe)
+      RecipeIngredient.where(recipe_id: recipe)
+   end
```

Comment:
Is this method necessary? It looks like you have a similar method in your `RecipeIngredient` model.